From 5bf732285682925375c998926c03116be6fa6d6c Mon Sep 17 00:00:00 2001
From: John Yu <byu@ncsu.edu>
Date: Thu, 29 Feb 2024 12:22:39 -0500
Subject: [PATCH] use configparser instead of toml

---
 run_ngboost.py | 22 ++++++++++++++++++----
 1 file changed, 18 insertions(+), 4 deletions(-)

diff --git a/run_ngboost.py b/run_ngboost.py
index 37b3043..34ed1b8 100755
--- a/run_ngboost.py
+++ b/run_ngboost.py
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-import tomllib
+from configparser import ConfigParser
 from argparse import ArgumentParser
 from pathlib import Path
 from time import time
@@ -22,14 +22,28 @@ args = parser.parse_args()
 args.predict_mean.parent.mkdir(parents=True, exist_ok=True)
 args.predict_std.parent.mkdir(parents=True, exist_ok=True)
 
-with open(args.config, 'rb') as config_file:
-    config = tomllib.load(config_file)
+config_parser = ConfigParser()
+with open(args.config, 'r') as config_file:
+    config_parser.read_file(config_file)
+
+base_params = {
+    'max_n_splits': int(config_parser['model']['max_n_splits']),
+    'min_partition_size': int(config_parser['model']['min_partition_size']),
+    'min_impurity_drop': float(config_parser['model']['min_impurity_drop']),
+    'batch_size': int(config_parser['model']['batch_size']),
+    'n_epochs': int(config_parser['model']['n_epochs']),
+}
+
+ensemble_params = {
+    'n_stages': int(config_parser['ensemble']['n_stages']),
+    'learn_rate': float(config_parser['ensemble']['learn_rate']),
+}
 
 train_feature = pd.read_csv(args.train_feature, index_col=['date'])
 train_target = pd.read_csv(args.train_target, index_col=['date'])
 test_feature = pd.read_csv(args.test_feature, index_col=['date'])
 
-model = NGBoost(base_params=config['model'], **config['ensemble'])
+model = NGBoost(base_params=base_params, **ensemble_params)
 
 train_start = time()
 model.fit(train_feature, train_target)
-- 
2.34.1

