From e149157c3fe73606267a0faa41bd4f89d29f52db Mon Sep 17 00:00:00 2001
From: John Yu <byu@ncsu.edu>
Date: Sun, 25 Feb 2024 18:29:20 -0500
Subject: [PATCH 3/3] disable plot tree

---
 run_model.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/run_model.py b/run_model.py
index becc119..b9fd001 100755
--- a/run_model.py
+++ b/run_model.py
@@ -57,11 +57,6 @@ predict = pd.DataFrame(predict,
 
 predict.to_csv(args.predict)
 
-if args.plot_tree is not None:
-    args.plot_tree.parent.mkdir(parents=True, exist_ok=True)
-    tree_plot = model.visualize(column_names=train_feature.columns)
-    tree_plot.draw(args.plot_tree)
-
 train_time = train_end - train_start
 test_time = test_end - test_start
 
-- 
2.34.1

